jetstream/config.py

```diff
@@ -196,9 +196,11 @@ def __getattr__(proxy, name):
 
     @property
     def features(self) -> List[nimbus.Feature]:
-        return [
-            self.feature_resolver.resolve(slug) for slug in self.experimenter_experiment.features
-        ]
+        return {
+            self.feature_resolver.resolve(branch.feature.featureId)
+            for branch in self.experimenter_experiment.branches
+            if branch.feature and branch.feature.enabled
+        }
```

Comment:
Looks like feature are specific to branches based on the Nimbus specs. Currently, we convert features to `Summary` and don't have anything in place to calculate a feature only for a specific branch. So I guess we'll continue to calculate features for all branches here?

Reply:
Ah, `features` were renamed to `probesets`

Reply:
Features as we know them got renamed to probeSets; the features associated with branches are a new concept that tell the client which surfaces within the browser are being configured. I think we don't have to worry about the features associated with branches -- just the probeSets associated with the experiment, which is what we call a Feature everywhere in Jetstream today.